index.html

```diff
@@ -668,6 +681,110 @@ <h2 id="constraints">Unconstrained Display Surface Selection</h2>
+        The user agent MUST NOT change the [=display surface=]
+        associated with a [=capture-session=] unless the user has
+        explicitly indicated that they want this change to be
+        performed by interacting with user agent or operating system.
+
+        If the user indicates a change of [=display surface=], the
+        user agent MUST run the following steps:
+      </p>
+
+
+      <ol>
+        <li>
+          <p>If the [=capture-session=] does not have an associated
+            {{CaptureController}}, <var>controller</var>, abort these steps.</p>
+        </li>
+
+
+        <li>
+          <p>If <var>controller.{{CaptureController/[[DisplaySurfaceChangeCallback]]}}</var>
+            is null, abort these steps.</p>
+        </li>
+
+
+        <li>
+          <p>Stop delivering frames from all sources in the
+            [=capture-source-set=] associated with the [=capture-session=].
+          </p>
+        </li>
+
+
+        <li>
+          <p>Let <var>newSources</var> be the set of sources selected by the
+            user when the user chose a new [=display surface=].</p>
+
+
+          <p>
+            <var>newSources</var> MUST fulfill the same requirements as the
+            [=capture-source-set=] formed when {{MediaDevices/getDisplayMedia}}
+            is called.</p>
+        </li>
+
+
+        <li>
+          <p>Queue a task to:</p>
+          <ol>
+            <li>
+              <p>
+                Set the [=capture-source-set=]
+                associated with the [=capture-session=] to <var>newSources</var>.
+              </p>
+            </li>
+
+
+            <li>
+              <p>Let <var>stream</var> be a new {{MediaStream}} with a new
+                {{MediaStreamTrack}} for each source in <var>newSources</var>.
+            </li>
+
+
+            <li>
+              <p>Run the [=ApplyConstraints algorithm=] on all tracks in
+                <var>stream</var> with the appropriate constraints. Should
+                this fail, abort these steps.</p>
+            </li>
+
+
+            <li>
+              <p>Set
+                <var>controller</var>.{{CaptureController/[[Source]]}} to
+                the <var>stream</var>'s video track's
+                <a data-cite="GETUSERMEDIA#dfn-source-0">[[\Source]]</a>.</p>
+            </li>
+
+
+            <li>
+              <p>Set
+                <var>controller</var>.{{CaptureController/[[DisplaySurfaceType]]}}
+                to the <var>stream</var>'s video track's
+                {{DisplayCaptureSurfaceType}}.</p>
+            </li>
+
+
+            <li>
+              <p>Invoke
+                <var>controller</var>.{{CaptureController/[[DisplaySurfaceChangeCallback]]}}
+                with <var>stream</var> as the argument.</p>
+            </li>
+          </ol>
+        </li>
+
+        <li>
+          <p>Queue tasks to end all tracks connected to sources in the
+            [=capture-source-set=] associated with the [=capture-session=].
```

Comment:
I think the default should be to leave the tracks alive, the future option being for the web page to ask for tracks to be ended.
The principle is that a web page can easily implement ending of the tracks, but it cannot implement the reverse.